### ISSUE TITLE
Ruby version must match what is in .ruby-version file

### ISSUE THREAD

**Issue by StrangeWill** [chadev/chatech-jobs#45]
Will have to look in to how Heroku uses this, it's pretty frustrating.

**Comment by t-richards** [comment]
@StrangeWill: When you specify `ruby X.Y.Z` in your `Gemfile`, Heroku will use that specific Ruby version while building/running your application. In this particular project the `.ruby-version` file contains the actual version number. This is nice because it allows you to do things like this:

```bash
$ echo "2.2.4" > .ruby-version
$ git update-index --assume-unchanged .ruby-version
```

Now you can use your desired version of Ruby without any issues (hopefully)!

**Comment by StrangeWill** [comment]
@t-richards Yeah if that's required then I'll drop it in the readme... I was just wondering if there was a less disruptive approach.
